test/factories/facts_related.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :fact_name do
     sequence(:name) {|n| "fact#{n}" }
+    type 'PuppetFactName'
```

Comment:
@lizagilman I would suggest to try without this line, as it turns out break many other tests.